ansible/roles_ocp_workloads/ocp4_workload_nginxplus/defaults/main.yml

```diff
@@ -7,7 +7,7 @@ rebuild_k8s_venv: false
 private_registry_url: "quay.io/redhat-gpst"
 image_name: "rhpds-nginxplus:latest"
 
-nginx_namespace: nginx-ingress-test
+nginx_namespace: nginx-ingress
```

Comment:
Can you please prefix variable names with the name of the workload.
E.g. ocp4_workload_nginxplus_nginx_namespace.
This avoids potential clashes with other workloads.

Reply:
Yes, will work on that today.  thx